### ISSUE TITLE
HueDynamic Issues

### ISSUE THREAD

**Issue by Pronown** [hass-emulated-hue/core#83]
**Describe the bug**

I was using the add-on just fine with HueDynamic to control my lights. Now the bridge is found and links, but will not fully pair. They were playing beautifully together last night, but now the app just says that there is an error connecting to the bridge.

**Steps to Reproduce**
1. Open HueDynamic
2. Pair Hub (either way)
3. Use push button notification
4. App pairs
5. Press Bridge Settings
6. Check Version

**Expected behavior**
Previously it showed a version and hub name on HueDynamic Settings and the app functioned perfectly. I was able to control lights and set disco functions. Now instead it says "There are problems connecting to the bridge. Please check your network." and does not show a version number, nor hub name.

Expected behavior for the app to 
**Logs**

```
2021-01-18 19:00:58,096 INFO  emulated_hue.config -- Link request detected - Use the Homeassistant frontend to confirm this link request.
2021-01-18 19:01:33,509 WARNING emulated_hue.api -- Invalid/unknown request: <Request DELETE /api/11285942-c420-4b0c-a12f-7a505fa1190d/config/whitelist/11285942-c420-4b0c-a12f-7a505fa1190d >
2021-01-18 19:02:08,358 INFO  emulated_hue.config -- Link mode is enabled for the next 5 minutes.
2021-01-18 19:07:08,362 INFO  emulated_hue.config -- Link mode is disabled.
2021-01-18 19:15:02,295 INFO  emulated_hue.config -- Link request detected - Use the Homeassistant frontend to confirm this link request.
2021-01-18 19:15:09,430 INFO  emulated_hue.config -- Link mode is enabled for the next 5 minutes.
2021-01-18 19:15:09,629 INFO  emulated_hue.api -- Client hueDynamic1#Android10.0.18446 registered
```

**Hardware Info:**
 - Architecture (Ex: x86, ARM): armv7l
 - Addon version (Ex: stable or dev): Stable
    - Value of "tag_commit_or_branch" if using dev: 
 - OS (Ex: Windows 10, Ubuntu Buster): 
    - Home Assistant: Linux 5.4.79-v7
    -  HueDynamic: Windows 10 and Android
 - Hardware (Ex: PC, Raspberry Pi 1/2/3/Zero): RPi3B+

**Additional context**

I figured I would add that I am also having trouble with Hue Sync as is in another bug.
Thank you for your time and all you do!

**Comment by alexyao2015** [comment]
Seems to be working fine for me? I would suggest you update to the latest version and delete your config file if still having issues.

**Comment by Pronown** [comment]
Oddly, it happens at random. Sometimes with just the android version, sometimes with just the windows version, or both. I've noticed that once it starts working, it works until the app is closed or the pc is shut down or restarted.
I can actually see the version and Home Assistant name now, version shows as 1941132080.
In this case, I just tested it now and it worked for a brief moment then stopped. I will try the config and update.
